src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -418,22 +504,24 @@ def test_start_job_status_loop__cell_ids(self):
                     assert self.jc._running_lookup_loop is False
                     assert self.jc._lookup_timer is None
 
-    @mock.patch(CLIENTS, get_failing_mock_client)
-    def test_start_job_status_loop__initialise_jobs_error(self):
-        # check_workspace_jobs throws an EEServerError
-        self.jc.start_job_status_loop(init_jobs=True)
-        assert self.jc._comm.last_message == {
-            "msg_type": ERROR,
-            "content": {
-                "code": -32000,
-                "error": "Unable to get initial jobs list",
-                "message": "check_workspace_jobs failed",
-                "name": "JSONRPCError",
-                "request": "jc.start_job_status_loop",
-                "source": "ee2",
-            },
-        }
-        assert self.jc._running_lookup_loop is False
+    @mock.patch(CLIENTS, get_mock_client)
+    def test_start_job_status_loop__initialise_jobs_error(self) -> None:
+        # patch all client calls to return clients that return errors
+        with mock.patch(CLIENTS, side_effect=get_failing_mock_client):
```

Comment:
changed this test to be more like a real situation -- JobComm and JobManager are fine during initialisation and throw no errors, but during the actual test, there's a communication error.